_data/toc.yaml

```diff
@@ -1441,12 +1441,10 @@ manuals:
   - sectiontitle: Install Compose
     section:
       - path: /compose/install/
-        title: Install Docker Compose 
-      - path: /compose/install/compose-desktop/
-        title: Install Compose plugin through Desktop 
-      - path: /compose/install/compose-plugin/
-        title: Install Docker Compose Plugin 
-      - path: /compose/install/uninstall/ 
+        title: Install Docker Compose
+      - path: /compose/install/compose-linux/
+        title: Install on Linux
```

Comment:
I'm not sure how this affects SEO, the keywords in the front matter are there to optimize searched. I just thought there was a lot of context repetition and, since, there is not a lot of real-estate in the TOC (panel on the left) and the titles wrap after a couple of words I thought the TOC looked cleaner that way. I am also fine either way. Your call.

Reply:
Sure, let's go with the shorter title.